index.js

```diff
@@ -1,5 +1,69 @@
 'use strict';
 
+var superagent = require('superagent');
+
+global.D2LAccessTokenExpiresAt = 0;
```

Comment:
That could work too - it would have the same concerns as #15 - ideally we'd use a storage library to fall back to a global (but then that fails for same-origin iframes, but in a way that could technically be worked around by reaching up through the frame repeatedly...)

I can change it to use localStorage for now.